pkg/connection/connection.go

```diff
@@ -0,0 +1,10 @@
+package connection
+
+import (
+	"k8s.io/client-go/kubernetes"
+)
+
+func TestKubeClientConnection(client *kubernetes.Clientset) error {
```

Comment:
Nit: in general I think golang discourage function with test prefix because that's their conversion for test method.

Reply:
You're right. Oversight